src/apps/core/views/monster-country-selector-item.html

```diff
@@ -0,0 +1,5 @@
+<span class="monster-country-item-label">{{{label}}}</span>
+{{#if code}}
+	{{!-- A background image was used instead of the img element to avoid displaying the broken image placeholder for unexistent flag images --}}
+	<span class="monster-country-item-flag" style="background-image:url(css/assets/flags/24/{{code}}.png)"></span>
+{{/if}}
```

Comment:
Unless the mockup says otherwise, the flag should be place before the country name to keep them aligned (since they are the same size) vertically. It will make it much more easier to parse visually.

As for using `background-image` instead of an `img` tag, that works but you will most likely have to create a container for your flag with the same width/height and always displayed to act as a placeholder.

The second solution would be to have a default flag image (not sure if we have one) and use an `img` tag like so:
```html
<img src="path/to/original/image.png" onerror="path/to/fallback/image.png">
```

This can also be achieved with CSS only:
```html
<img src="path/to/original/image.png">
```
```scss
img {
	position: relative;

	&:before {
		content: ' ';
		display: block;
		position: absolute;
		top: 0;
		left: 0;
		height: 24px;
		width: 24px;
		background: transparent url('path/to/fallback/image.png');
	}
}
```

But then again, it all depends if we have a fallback image or not.

Reply:
Indeed, there is a fallback image :open_mouth: 

There is not a mockup for the actual dropdown list, only for the selected item, and the flag is placed to the right for the latter:

![image](https://user-images.githubusercontent.com/893877/67431589-d561b180-f5a1-11e9-86d2-bc3534940662.png)

However I agree with you about displaying the image to the left. Do you think it would look odd if I display the flag to the left for the list items, and to the right for the selected item?

Reply:
I think the flag should precede the country label in any case for visual consistency.

As you said, the mockup does not come with an expanded dropdown so I assume that use case was not taken into account during design.